pallets/pallet-web3-names/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
+authors = ["KILT <info@kilt.io>"]
+description = "Unique Web3 nicknames for KILT DIDs."
+edition = "2021"
+name = "pallet-web3-names"
+repository = "https://github.com/KILTprotocol/mashnet-node"
+version = "1.3.1"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dev-dependencies]
+runtime-common = {path = "../../runtimes/common"}
```

Comment:
No runtime-common dependencies in pallets please.

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/314/commits/697e97815c5432bef5f0c6fc77c0e831f4d18a64